master/buildbot/worker/base.py

```diff
@@ -296,7 +297,10 @@ def stopService(self):
 
     def startMissingTimer(self):
         if self.notify_on_missing and self.missing_timeout and self.parent:
-            self.stopMissingTimer()  # in case it's already running
+            try:
+                self.stopMissingTimer()  # in case it's already running
+            except error.AlreadyCalled:
```

Comment:
I think here, we are pushing the dust under the carpet.
I would prefer that there is more investigation. Can you reproduce with latest code?

I would prefer if the fix is in stopMissingTimer.

    def stopMissingTimer(self):
        if self.missing_timer:
            if self.missing_timer.active():
                self.missing_timer.cancel()
            self.missing_timer = None